strangerstudios/paid-memberships-pro
# Refunds for Stripe & PayPal Express

### All Submissions:






### Changes proposed in this Pull Request:

Allows you to refund an order in three different places. Specific to Stripe & PayPal Express


### How to test the changes in this Pull Request:

1. Navigate to Memberships > Orders and click on the Refund button or
2. Navigate to Memberships > Orders > Edit and click on the Refund button or
3. Edit a user profile, change the level and check the checkbox to 'Refund this user's most recent order'

### Other information:




### Changelog entry

One-click refunds supported for Stripe and PayPal Express orders
